spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -1,10 +1,10 @@
 require 'mutant/integration/rspec'
 
 RSpec.describe Mutant::Integration::Rspec do
-  let(:object) { described_class.new }
+  let(:object) { described_class.new(Mutant::Config::DEFAULT) }
 
-  let(:options)       { double('options')                           }
-  let(:runner)        { double('runner')                            }
+  let(:options) { double('options') }
```

Comment:
What do you think about using `instance_double` instead? It has stronger constraints about how it can be stubbed.